tests/integration/entity/helpers/requestSender.ts

```diff
@@ -0,0 +1,20 @@
+import * as supertest from 'supertest';
+import { Application } from 'express';
+
+import { container } from 'tsyringe';
+import { ServerBuilder } from '../../../../src/serverBuilder';
+
+export function getApp(): Application {
+  const builder = container.resolve<ServerBuilder>(ServerBuilder);
+  return builder.build();
+}
+
+export function getMockedRepoApp(repo: unknown): Application {
+  container.register('EntityRepository', { useValue: repo });
+  const builder = container.resolve<ServerBuilder>(ServerBuilder);
+  return builder.build();
+}
+
+export async function createEntity(app: Application, entity: { osmId?: unknown; externalId?: unknown }): Promise<supertest.Response> {
+  return supertest.agent(app).post('/entity').set('Content-Type', 'application/json').send(entity);
+}
```

Comment:
why did you decide to get the app as a parameter not set the app as this file module variable?

Reply:
This way the code is not tightly coupled with one express app, and its possible for example, to have one app with sqlite DB, and another with mocked DB.